version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.23.1-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.24.0-SNAPSHOT"
```

Comment:
please don't change the version as part of this PR.

Reply:
Woops! did not mean to push that, haha. Just using it locally.